test/node-rpc-test.js

```diff
@@ -85,9 +104,11 @@ describe('RPC', function() {
       for (const property of properties)
         assert(property in info);
 
-      assert.deepEqual(chain.height, info.height);
-      assert.deepEqual(chain.tip, info.hash);
-      assert.equal(info.bits, '207fffff');
+      assert.deepEqual(node.chain.tip.height, info.height);
+      assert.deepEqual(node.chain.tip.bits, parseInt(info.bits, 16));
+      assert.deepEqual(util.revHex(node.chain.tip.merkleRoot), info.merkleroot);
+      assert.deepEqual(util.revHex(node.chain.tip.hash), info.hash);
+      assert.equal(node.chain.tip.version, info.version);
```

Comment:
OK these are great, good job! Personally I would use `strictEqual` for these (I use `deepStrictEqual` for example when checking all elements of an array or object) but I'm actually not sure if that makes a difference here. They are both implemented in this dependency if you are curious (maybe you wanna teach me something?) https://github.com/chjj/bsert/blob/master/lib/assert.js